kurtosis-devnet/pkg/kurtosis/api/enclave/enclave.go

```diff
@@ -53,3 +75,46 @@ func (mgr *KurtosisEnclaveManager) GetEnclave(ctx context.Context, enclave strin
 
 	return enclaveCtx, nil
 }
+
+func (mgr *KurtosisEnclaveManager) Autofix(ctx context.Context, enclave string) error {
+	fmt.Printf("Autofixing enclave '%s'\n", enclave)
+	status, err := mgr.kurtosisCtx.GetEnclaveStatus(ctx, enclave)
+	if err != nil {
+		// Means the enclave doesn't exist, so we're good
+		return nil
+	}
+	switch status {
+	case interfaces.EnclaveStatusRunning:
+		return nil
+	case interfaces.EnclaveStatusStopped:
+		fallthrough
+	case interfaces.EnclaveStatusEmpty:
+		// Remove the enclave
+		err := mgr.kurtosisCtx.DestroyEnclave(ctx, enclave)
+		if err != nil {
+			return fmt.Errorf("failed to destroy enclave: %w", err)
+		}
+		fmt.Printf("Destroyed enclave: %s\n", enclave)
+		err = mgr.dockerMgr.DestroyDockerResources(ctx, enclave)
```

Comment:
direct calls to docker mgr are not quite correct, as we don't know for sure that Kurtosis is running on Docker. So we should inspect the engine to figure out what backend it's currently using.

Also, shouldn't we try to clean this up even if the DestroyEnclave failed?